SalesforceSDK/Salesforce.SDK.Universal/Source/Hybrid/HybridMainPage.cs

```diff
@@ -235,7 +240,13 @@ private void LoadUri(Uri uri)
             browser.NavigationStarting += onBrowserNavigationStarting;
             if (browser.Source == null)
             {
-                browser.Navigate(uri);
+                try
+                {
+                    browser.Navigate(uri);
+
+                }
+                catch (ArgumentException)
```

Comment:
Do we need to do anything in the catch block? What happens if browser.Navigate fails?